src/Relude/Extra/Newtype.hs

```diff
@@ -0,0 +1,49 @@
+{-# LANGUAGE ScopedTypeVariables #-}
+
+{- | Functions to ease work with newtypes.
+-}
+
+module Relude.Extra.Newtype
+       ( un
+       , wrap
+       , under
+       ) where
+
+import Relude
+
+-- $setup
+-- >>> :set -XTypeApplications
+
+{- | Unwraps value from @newtype@.
+
+>>> newtype Size = Size Int deriving Show
+>>> un @Int (Size 5)
+5
+>>> un (Size 5) == length ['a', 'x', 'b']
+False
+-}
+un :: forall b a . Coercible a b => a -> b
```

Comment:
Wouldn't `unwrap` be a more appropriate name here, since you've also added a `wrap` function that is doing the opposite thing?

Reply:
@ghallak That's a possible option. But I would like to have shorter name. Usually you define `newtype`s like this:

```haskell
newtype Size = Size { unSize :: Int }
```

So `un (Size 3)` looks pretty nice. And `unwrap` has the same length as `unSize`. No much benefit. Also, in some cases you need to use type applications which will make your code even bigger. So `un @Int` is okay if you want to be more explicit about your intention.

It's arguable, what is better, `un` or `unwrap`. But I would like to give shorter version a try and see how it goes.